### ISSUE TITLE
enrolled_following_domain untranslated in NL locale

### ISSUE THREAD

**Issue by thijskh** [SURFnet/tiqr-client-ios#24]
After enrolling in NL locale I get the following screen. `enrolled_following_domain` seems to be missing from the nl translation file. This should be synced with the EN one and a translation added.

![img_2900](https://cloud.githubusercontent.com/assets/3808792/21450796/46606840-c8fa-11e6-8886-2027954867bd.PNG)

**Comment by erikvdwal** [comment]
@thijskh I can only find an English translation for that key, meaning the translation is missing (and defaults to `enrolled_following_domain`) for all other localizations. It seems to have been added last year as part of a design update.

However, I see there's also a key named `enroll_following_domain`, which has a similar localized value and _is_ present for every localization. Could it be that the wrong key was accidentally used? The value for this key also seems to make more sense, because it's displayed before confirms the account.

```
/* You will enroll to the following domain */
"enroll_following_domain" = "You will enrol to the following domain:";

/* You are enrolled for the following domain: */
"enrolled_following_domain" = "You are enrolled for the following domain:";
```

**Comment by thijskh** [comment]
@joostd Can you add this in the translation portal?

**Comment by thijskh** [comment]
Likely solved by 0c4f00d6affbd61a0a6e2d8d1a50928bcad87b21
